.github/workflows/synth-test.yml

```diff
@@ -0,0 +1,29 @@
+
+name: synth-test
```

Comment:
as-is currently this is not exclusively testing synth, it does run unit tests as well.

Reply:
Good point.  I have updated the PR accordingly.